File: chatwoot/typescript-node/src/GetPortalExample.ts
import * as fs from 'fs';
import api from "chatwoot_client"
import models from "chatwoot_client"

const apiCaller = new api.HelpCenterApi();
apiCaller.setApiKey(api.HelpCenterApiApiKeys.userApiKey, "USER_API_KEY");

apiCaller.getPortal(
  0, // accountId
).then(response => {
  console.log(response.body);
}).catch(error => {
  console.log("Exception when calling HelpCenterApi#getPortal:");
  console.log(error.body);
});

File: chatwoot/kotlin/src/main/kotlin/oseg/chatwoot_examples/GetPortalExample.kt
package oseg.chatwoot_examples

import com.chatwoot.client.infrastructure.*
import com.chatwoot.client.apis.*
import com.chatwoot.client.models.*

import java.io.File
import java.time.LocalDate
import java.time.OffsetDateTime
import kotlin.collections.ArrayList
import kotlin.collections.List
import kotlin.collections.Map
import com.squareup.moshi.adapter

@ExperimentalStdlibApi
class GetPortalExample
{
    fun getPortal()
    {
        ApiClient.apiKey["api_access_token"] = "USER_API_KEY"

        try
        {
            val response = HelpCenterApi().getPortal(
                accountId = 0,
            )

            println(response)
        } catch (e: ClientException) {
            println("4xx response calling HelpCenterApi#getPortal")
            e.printStackTrace()
        } catch (e: ServerException) {
            println("5xx response calling HelpCenterApi#getPortal")
            e.printStackTrace()
        }
    }
}
